eng/readme-templates/Support.md

```diff
@@ -18,5 +18,5 @@
 {{ARGS["top-header"]}} License
 
 * [Microsoft Container Images Legal Notice](https://aka.ms/mcr/osslegalnotice): applies to all [.NET Framework container images]({{InsertTemplate("Url.md", [ "readme-host": ARGS["readme-host"], "repo": "dotnet/framework", "is-product-family": "true" ])}})
-* [Microsoft Software Supplemental License for Windows Container Base Image]({{InsertTemplate("Url.md", [ "readme-host": ARGS["readme-host"], "repo": "windows/servercore", ])}}): applies to all [.NET Framework container images]({{InsertTemplate("Url.md", [ "readme-host": ARGS["readme-host"], "repo": "dotnet/framework", "is-product-family": "true" ])}})
+* [Windows Base Image License](https://learn.microsoft.com/en-us/virtualization/windowscontainers/images-eula): applies to all [.NET Framework container images]({{InsertTemplate("Url.md", [ "readme-host": ARGS["readme-host"], "repo": "dotnet/framework", "is-product-family": "true" ])}})
```

Comment:
Remove the locale so it will default to the user's preferred locale.

```suggestion
* [Windows Base Image License](https://learn.microsoft.com/virtualization/windowscontainers/images-eula): applies to all [.NET Framework container images]({{InsertTemplate("Url.md", [ "readme-host": ARGS["readme-host"], "repo": "dotnet/framework", "is-product-family": "true" ])}})
```

Reply:
This also applies to several other places outside of this template.